### PR TITLE
Add nodeSelecetor flag for hypershift create and pass it through env variable for cluster-bot

### PR DESCRIPTION
**Pull request by enxebre**
/hold
Needs https://github.com/openshift/hypershift/pull/1759 and create the MachinePool on the Management cluster for the Nodes with that role.

### CODE REVIEW

**Review by vrutkovs — APPROVED** [review]
/lgtm

**Comment by sjenning** [comment]
waiting for a good nightly to promote for rehearse to pass
https://amd64.ocp.releases.ci.openshift.org/#4.12.0-0.nightly

File: ci-operator/step-registry/hypershift/aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yaml [hunk 1]
```diff
@@ -44,7 +52,7 @@ chain:
         --service-cidr=172.29.0.0/16 \
         --node-pool-replicas=${HYPERSHIFT_NODE_COUNT} \
         --release-image=${HYPERSHIFT_HC_RELEASE_IMAGE} \
-        --namespace=${HYPERSHIFT_NAMESPACE}
+        --namespace=${HYPERSHIFT_NAMESPACE} ${HYPERSHIFT_HC_NODE_SELECTOR_FLAG}
```
Comment (sjenning) [inline ci-operator/step-registry/hypershift/aws/setup-nested-management-cluster/hypershift-aws-setup-nested-management-cluster-chain.yaml:55]: nit: can we put this on a new line like all the other flags?

**Comment by sjenning** [comment]
required PR merged
/hold cancel

**Pull request closed without merging**